src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,298 @@
+package duke;
+
+import duke.tasks.*;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class Duke {
+    public static ArrayList<Task> tasks = new ArrayList<>();
+
+    public static void dividerLine() {
+        System.out.println("\t________________________________________\n");
+    }
+
+    public static void loadData() {
```

Comment:
This method seems to be a little long, consider extracting and refactoring parts of the code to make it shorter.